tool/grind.dart

```diff
@@ -0,0 +1,24 @@
+library polymer_interop.grind;
+
+import 'package:grinder/grinder.dart';
+
+const dartium = 'dartium';
```

Comment:
I just use this to quickly try different browsers (with autocompletion) but I'm okay with deleting it